software/ever-gauzy.yml

```
@@ -0,0 +1,19 @@
+name: "Ever Gauzy"
+website_url: "https://gauzy.co"
+source_code_url: "https://github.com/ever-co/ever-gauzy"
+description: "Open Business Management Platform (ERP/CRM/HRM/ATS/PM)"
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - Nodejs
+  - Javascript
+tags:
+  - Time Tracking
+  - Inventory Management
+  - Resource Planning
+  - Software Development - Project Management
+  - Task Management & To-do Lists
+  - Ticketing
+  - Calendar & Contacts
```

I would argue that this app is out of the scope for a couple of these categories so I think these are the best to define it.

```suggestion
tags:
  - Resource Planning
  - Task Management & To-do Lists
```

---

@Ki-er well, Time Tracking is actually huge part of the platform related to HRM... I.e. we are probably the only OSS that allows time tracking with screenshots etc. We also have some functionality related to Inventory Management, Project Management (tasks / sprints) and so on... Basically in some sense it's similar to Odoo where it's all-in-one platform for businesses, so it covers a lot of aspects... Can we keep those categories? If it's too wide, then let's use what you suggested, but please just add at least "Time Tracking" and "Inventory Management" to the list :)